docs/tutorial_notebooks/tutorial1/Lisa_Cluster.ipynb

```diff
@@ -205,7 +220,7 @@
     "\n",
     "If you work with a lot of data, or a larger dataset, it is advised to copy your data to the `/scratch` directory of the node. Otherwise, the read/write operation might become a bottleneck of your job. To do this, simply use your copy operation of choice (`cp`, `rsync`, ...), and copy the data to the directory `$TMPDIR`. You should add this command to your job file before calling `srun ...`. Remember to point to this data when you are running your code. If you have a dataset that can be downloaded, you can also directly download it to the scratch (can sometimes be faster than copying). In case you also write something on the scratch, you need to copy it back to your home directory before finishing the job. \n",
     "\n",
-    "**Edit Dec. 6, 2021**: Due to internal changes to the filesystem of Lisa, it is required to **use the scratch for any dataset** such as CIFAR10. In PyTorch, the CIFAR10 dataset is structured into multiple large batches (usually 5), and only that batch is loaded which is currently needed. This is why during training, it requires a lot of reading operations on the disk which can slow down your training and constitutes a challenge to the Lisa system when hundreds of students share the same filesystem. Hence, you have to use the scratch for such datasets. For most parts in the assignment, you can do this by specifying `--data_dir $TMPDIR` on the python command in your job file (check if the argument parser has this argument, otherwise you can add it yourself). This will download the dataset to the scratch and only load it from there. We recommend using this approach also for future course editions."
+    "**Edit Dec. 6, 2021**: Due to internal changes to the SURF filesystems, it is required to **use the scratch for any dataset** such as CIFAR10. In PyTorch, the CIFAR10 dataset is structured into multiple large batches (usually 5), and only that batch is loaded which is currently needed. This is why during training, it requires a lot of reading operations on the disk which can slow down your training and constitutes a challenge when hundreds of students share the same filesystem. Hence, you have to use the scratch for such datasets. For most parts in the assignment, you can do this by specifying `--data_dir $TMPDIR` on the python command in your job file (check if the argument parser has this argument, otherwise you can add it yourself). This will download the dataset to the scratch and only load it from there. We recommend using this approach also for future course editions."
```

Comment:
I have been in contact with SURF about this, and they told me that the best practice is a bit complex. Generally, reading many small files from the filesystem is not recommended, but the best practice ultimately depends on your dataset.

One option is to use nodes that have $TMPDIR pointing to local scratch disks (--constraint=scratch-node), which is explained further on the SURF wiki ([https://servicedesk.surf.nl/wiki/display/WIKI/Snellius+hardware+and+file+systems#Snelliushardwareandfilesystems-The/scratch-nodespace:trulynode-localscratch).](https://servicedesk.surf.nl/wiki/display/WIKI/Snellius+hardware+and+file+systems#Snelliushardwareandfilesystems-The/scratch-nodespace:trulynode-localscratch). "smartCard-inline")  I/O operations on small files should perform well on these nodes. However, since there is a limited number of such nodes available, I do not believe this is the best solution for DL1 specifically. - Another option is to use packed file formats ([https://servicedesk.surf.nl/wiki/display/WIKI/Best+Practice+for+Data+Formats+in+Deep+Learning).](https://servicedesk.surf.nl/wiki/display/WIKI/Best+Practice+for+Data+Formats+in+Deep+Learning). "‌") With packed file formats, performing I/O directly on the shared filesystem, e.g., on the 'regular' (non-local) scratch, is acceptable. This allows you to use it with any of the GPU nodes (including the ones that don't have local scratch) and with much larger datasets.

Overall, I think it is safest to inherit the best practice from last year (i.e., setting the --data_dir $TMPDIR flag in the training scripts by default).